x-pack/plugins/case/common/constants.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const APP_ID = 'case';
+
+/**
+ * Case routes
+ */
+
+export const CASE_URL = '/api/cases';
+export const CASE_SINGLE_URL = `${CASE_URL}/{case_id}`;
+export const CASE_CONFIGURE_URL = `${CASE_URL}/configure`;
+export const CASE_CONFIGURE_CONNECTORS_URL = `${CASE_CONFIGURE_URL}/connectors`;
+export const CASE_COMMENT_URL = `${CASE_SINGLE_URL}/comments`;
```

Comment:
I think this should be named `CASE_COMMENTS_URL` since it returns all of the comments for a single case